addons/mail/models/mail_message.py

```diff
@@ -1087,6 +1089,7 @@ def _cleanup_side_records(self):
         self.write({
             'starred_partner_ids': [(5, 0, 0)],
             'notification_ids': [(5, 0, 0)],
+            'pinned_at': None,
```

Comment:
Why that here? `pinned_at` is not a side record, just a regular field.

Reply:
This will indirectly clean the channel side record but I understand your point, I moved the write on `pinned_at` to the `_message_update_content` method.